core/style.css

```diff
@@ -106,6 +106,10 @@ i.large { font-size: 6rem; }
   color: #00c5ff;
 }
 
+.mac {
+  color: #ff0028;
```

Comment:
I recommend a lighter color, it doesn't have to match map layers (as other teams)

Reply:
I have to agree. This kind of red on blue is really hard to read.

Reply:
Red on blue is always pain to read, literally...
So I'm open to any suggestions.

Maybe we could just drop to white on blue...
Or if you find better red, just gimme the number 😉

And this change is supposed only for the info tab, right? Portal and link color on map is ok, isn't it?

Reply:
Personally I've been using this via Stylus:
```
    .mac {
	  color: #ff1030;
	  text-shadow: 1px 1px 3px black, 1px 1px 3px black;
	}
```
It is dark, but looks kind of cool and I got used to it.

Reply:
i've found `#ff2020` a bit better than the original..

If we want to add shadow, we should do it for all the factions, right?

Reply:
Actually you might want to add `text-shadow: none;` for other texts as some non-faction text would have shadow for that. I think the red with darker background looks kind of creepy and bloody. So as it should I think ;-)